src/views/PurchaseOrderDetail.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-back-button default-href="/" slot="start"></ion-back-button>
+        <ion-title>{{$t("Purchase Order Details")}}</ion-title>
+        <ion-buttons slot="end">
+          <ion-button @click="addProduct"><ion-icon :icon="add"/></ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content :fullscreen="true">
+      <div>
+        <ion-item lines="none">
+          <h1>{{$t("Purchase Order")}}: PO10291</h1>
+        </ion-item>
+        <ion-item>
```

Comment:
Please refer to ionic framework documentation for correct usage of ionic UI components
https://ionicframework.com/docs/api/input

Reply:
Implemented Ma'am.

Reply:
Please see it again

Reply:
This is not fixed yet